scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -494,38 +194,39 @@ task CreateFOFNs {
      }
 }
 
-task CreateImportTsvs {
+task LoadData {
   input {
     Array[File] input_vcfs
     Array[File] input_vcf_indexes
     Array[String] sample_names
     File interval_list
-    String output_directory
     File sample_map
     String? service_account_json_path
     String? drop_state
     Boolean? drop_state_includes_greater_than = false
 
-    Boolean call_cache_tsvs = false
     Boolean duplicate_check_passed
 
+    String project_id
+    String dataset_name
+
+    Boolean load_ref_ranges = true
+    Boolean load_pet = false
+    Boolean load_vet = true
+
     # runtime
     Int? preemptible_tries
     File? gatk_override
     String docker
 
-    String? for_testing_only
-    String run_uuid
-    Boolean load_pet
-    Boolean load_ref_ranges
   }
 
-  Int disk_size = if defined(drop_state) then 50 else 75
-
   String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
 
   meta {
-    description: "Creates a tsv file for import into BigQuery"
+    description: "Load data into BigQuery using the Write Api"
+
+    # TODO: is this right?  should this call cache?
```

Comment:
well, if one of the shards of `LoadData` failed for some reason that didn't mean the whole run was suspect, would we want to empty the table(s) in BigQuery and start over, or re-run and just do the failed shards?

Reply:
i *think* just redo the failed shards.  However, this is also obscured by the fact that the first thing we do is check "are you trying to load any samples that have already been loaded"… (in `CheckForDuplicateData`).  So in the scenario you describe above we wouldn't even get to the call-caching piece because we would fail on the `CheckForDuplicateData` check.  Thinking about it… in the scenario where you would want to call cache (it's already loaded, just call cache the loaddata step) we would always fail on the "is it loaded test"